src/components/RangeSlider/components/SingleThumb/SingleThumb.scss

```diff
@@ -93,19 +106,25 @@ $range-thumb-shadow-focus: 0 0 0 rem(1px) color('indigo');
     cursor: -webkit-grab;
     width: $range-thumb-size;
     height: $range-thumb-size;
-    border: border-width() solid color('sky', 'lighter');
+    border: border-width() solid
+      var(--p-action-interactive, color('sky', 'lighter'));
     border-radius: 50%;
-    background: linear-gradient(color('white'), color('sky', 'lighter'));
+    background: linear-gradient(
+      var(--p-action-interactive, color('white')),
+      var(--p-action-interactive, color('sky', 'lighter'))
+    );
     box-shadow: $range-thumb-shadow;
     appearance: none;
-    transition-property: border-color, box-shadow;
+    transition-property: border-color, box-shadow, margin-top, transform;
```

Comment:
Why do we need to transition the `margin-top`?

Reply:
We don't! Accidentally left this behind when we swapped from increasing the width/height of the thumb, to adjusting the scale.